packages/styling/src/interfaces/IFontStyles.ts

```diff
@@ -4,16 +4,16 @@ import { IRawStyle } from '../interfaces/index';
  * UI Fabric font set.
  */
 export interface IFontStyles {
```

Comment:
Does theme ever have any of these values as undefined? Don't we always put a default there?